docs/advanced/additional-requests.rst

```diff
@@ -604,7 +604,7 @@ from the previous subsection named: :ref:`httpclient-get-example`.
             except web_poet.exceptions.HttpResponseError as err:
                 logger.warning(
                     f"Received a {err.response.status} response status for product ID "
-                    f"'{item['product_id']}' from this URL: {err.request.url}"
+                    f"'{item['product_id']}' from this URL: {str(err.request.url)}"
```

Comment:
I think this is not needed (see https://github.com/scrapinghub/web-poet/pull/42#discussion_r888579354)

Reply:
Good catch! Updated in https://github.com/scrapinghub/web-poet/pull/48/commits/9287744af2b4110a91e447d1381efca7b85a1ea2